apps/menu-bar/modules/progress-indicator/ios/ProgressIndicatorModule.swift

```diff
@@ -0,0 +1,17 @@
+import ExpoModulesCore
+
+public class ProgressIndicatorModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ProgressIndicator")
+
+    View(ProgressIndicatorView.self) {
+      Prop("indeterminate") { (view, isIndeterminate: Bool) in
+        view.setIndeterminate(isIndeterminate)
+      }
```

Comment:
```suggestion
      Prop("indeterminate") { (view, isIndeterminate: Bool?) in
        if let isIndeterminate {
          view.setIndeterminate(isIndeterminate)
        } else {
          view.setIndeterminate(false)
        }
      }
```
We should probably make the props optional because if they are removed we'll get nil here. Probably not a big deal as it's only us using it.